index.js

```diff
@@ -224,7 +224,7 @@ async function run() {
     }
 
     const forceNewDeployInput = core.getInput('force-new-deployment', { required: false }) || false;
-    const forceNewDeployment = forceNewDeployInput != undefined && (forceNewDeployInput.toLowerCase === 'true' || forceNewDeployInput);
+    const forceNewDeployment = forceNewDeployInput != undefined && (forceNewDeployInput.toLowerCase() === 'true' || forceNewDeployInput);
```

Comment:
To be honest I'd keep it simple and only accept boolean values here. This is what AWS SDK does and would be consistent with **wait-for-service-stability** param management

Reply:
```suggestion
    const forceNewDeployment = core.getInput('force-new-deployment', { required: false }) || false;
```